src/services/refactors/inlineVariable.ts

```diff
@@ -244,5 +245,9 @@ function getReplacementExpression(reference: Node, replacement: Expression): Exp
         return factory.createParenthesizedExpression(replacement);
     }
 
+    if (isIdentifier(reference) && isShorthandPropertyAssignment(parent)) {

```

Comment:
To keep the nice documentation convention in this file, a comment here would be nice ;)